stubs/Modules/Base/App/Traits/Seoable.php

```diff
@@ -52,8 +52,8 @@ public static function seoFields(): Group
                         ])),
 
                     Forms\Components\Toggle::make('noindex')
-                        ->label(__('Allow index'))
-                        ->default(true),
+                        ->label(__('Don\'t allow index'))
```

Comment:
Op deze manier vraag je om een dubbele negatief

Wil je indexeren NIET toestaan? antwoord: Ja 

is verwarrend.

Ik zou het label gelijk laten (Allow index)
en dan een ->dehydrateStateUsing() en ->hydrateStateUsing toevoegen die de boolean om draait.